app/pages/sections/order-items/additional-services/order-item/controller.test.js

```diff
@@ -109,4 +112,61 @@ describe('additional-services order-item controller', () => {
       });
     });
   });
+
+  describe('getOrderItemErrorPageContext', () => {
+    afterEach(() => {
+      jest.resetAllMocks();
+    });
+
+    it('should call getSelectedPriceManifest with the correct params', async () => {
+      await getOrderItemErrorPageContext({
+        orderId: 'order-1',
+        orderItemType,
+        itemName: 'item-name',
+        selectedPriceId: 'some-price-id',
+        selectedPrice,
+        formData: { quantity: '100' },
+        validationErrors: {},
+      });
+
+      expect(getSelectedPriceManifest.getSelectedPriceManifest.mock.calls.length).toEqual(1);
+      expect(getSelectedPriceManifest.getSelectedPriceManifest).toHaveBeenCalledWith({
+        orderItemType,
+        provisioningType: selectedPrice.provisioningType,
+        type: selectedPrice.type,
+      });
+    });
+
+    it('should call getOrderItemErrorPageContext with the correct params', async () => {
+      const selectedPriceManifest = { description: 'fake manifest' };
+      getSelectedPriceManifest.getSelectedPriceManifest.mockReturnValue(selectedPriceManifest);
+
+      await getOrderItemErrorPageContext({
+        orderId: 'order-1',
+        itemName: 'item-name',
+        selectedPriceId: 'some-price-id',
+        selectedPrice,
+        formData: {
+          quantity: '100  ',
+          price: '  0.1',
+          selectEstimationPeriod: undefined,
+        },
+      });
+
+      expect(contextCreator.getErrorContext.mock.calls.length).toEqual(1);
+      expect(contextCreator.getErrorContext).toHaveBeenCalledWith({
+        commonManifest: { title: 'fake manifest' },
+        selectedPriceManifest,
+        orderId: 'order-1',
+        itemName: 'item-name',
+        selectedPriceId: 'some-price-id',
+        selectedPrice,
+        formData: {
+          quantity: '100',
+          price: '0.1',
+          selectEstimationPeriod: undefined,
```

Comment:
I don't know if this was deliberate but you don't have to put this it check if a key is undefined